### ISSUE TITLE
[bug] Finding from Id Games doesn't work with large amount of missing zips.

### ISSUE THREAD

**Issue by CortexReaver** [nstlaurent/DoomLauncher#266]
Seems like it tries to download it, but only creates a lot of "dummies" with 0 kb size in a Temp folder.
![24-01-2023 005859](https://user-images.githubusercontent.com/34350690/214159531-dceb8f8c-d028-4554-89d2-4535aa35985f.png)
(I've tried with a "real" temp folder and a redirected one - same result)

**Comment by nstlaurent** [comment]
Didn't you set yours up to use the ram drive as the temp folder? Did you try it without doing that? It's only going to cause issues.

**Comment by CortexReaver** [comment]
> Didn't you set yours up to use the ram drive as the temp folder? Did you try it without doing that? It's only going to cause issues.

> (I've tried with a "real" temp folder and a redirected one - same result)

**Comment by CortexReaver** [comment]
Also what I've tried to move mods folder to another drive completely and try syncing again - same.

**Comment by CortexReaver** [comment]
![24-01-2023 231820](https://user-images.githubusercontent.com/34350690/214399591-2266e420-c170-4a86-86c4-42a2834e7006.png)
Tried all available mirror servers - nothing works.

**Comment by CortexReaver** [comment]
By the way, your .msi installer creates a link on a desktop not on an installed program, but on installer itself.

**Comment by CortexReaver** [comment]
Downloading is working in Win 10, not Win7 (no more annoying error). Which is understandable, Windows 7 is an old unsupported piece of shit and I can take that. But is better to exclude it from the description on main page (well, its half-working with most of the important features).
Also, "finding in Id Games" doesn't copies files from "Temp" folder (don't start nitpicking on the RAM disk here, I'm tested all possible variations, including LEGIT TEMP FOLDER, reinstalling Doom Launcher and Net Framework several times). So, If anyone's wondering - If you have a lot of missing zips, you have first to wait until they all be downloaded (and there is no such indication for mass downloading, you can only judge by the size of zip "dummies" - if it has size more than zero, then its ready) and THEN copy all of them BY HAND in your mods folder from TEMP FOLDER.

**Comment by nstlaurent** [comment]
The error posted previously could happen if your clock is way off. Can you browse to any https site on that machine? I don't think it has anything to do with Windows 7.
 
The download wasn't designed to display a large of amount of files at once. I was assuming that the information you provided was correct in that all the files had downloaded.

I can fix the sync to ignore case.

**Comment by CortexReaver** [comment]
No, my clock is fine. Re-synced it right now and checked it again - same error in Win 7. If I cannot use any https site, how did you think I wrote all of this?

**Comment by CortexReaver** [comment]
Also it can't find some particular files, like these wads for example:
![25-01-2023 033105](https://user-images.githubusercontent.com/34350690/214452932-d6e22c1b-0ca0-4906-a8cc-2f71df809e5c.jpg)

But they are definitely exist on a Id Games: 
https://www.doomworld.com/idgames/levels/doom2/megawads/ur
https://www.doomworld.com/idgames/levels/doom2/Ports/megawads/white2
